tests/scripts/list-symbols.sh

```diff
@@ -14,8 +14,20 @@ fi
 
 cp include/mbedtls/config.h include/mbedtls/config.h.bak
 scripts/config.pl full
-CFLAGS=-fno-asynchronous-unwind-tables make clean lib >/dev/null 2>&1
+make_ret=
+CFLAGS=-fno-asynchronous-unwind-tables make clean lib \
+      >list-symbols.make.log 2>&1 ||
+  {
+    make_ret=$?
+    echo "Build failure: CFLAGS=-fno-asynchronous-unwind-tables make clean lib"
+    cat list-symbols.make.log >&2
+  }
+rm list-symbols.make.log
```

Comment:
With `set -e`, this `rm` won't happen if an error happens and a log will be left behind, right? Just an observation, but I think OK for a dumb script if so.

Reply:
No: since `make` is part of `make || { … }`, the script doesn't halt if `make` fails. What happens if `make` fails is that the `{ … }` block is executed.